src/main/core-api/java/com/mysql/cj/conf/PropertyKey.java

```diff
@@ -301,6 +301,10 @@ public enum PropertyKey {
     failureDetectionCount("failureDetectionCount", true),
     monitorDisposalTime("monitorDisposalTime", true),
 
+    // AWS Secrets Manager Plugin
+    secretsManagerSecretId("secretsManagerSecretId", true),
```

Comment:
We can't define new properties in this class. As soon as plugin class be extracted to another repo there won't be available for modifications. Such properties need to be defined in plugin class.